tests/api/features/steps/common_steps.py

```diff
@@ -3,7 +3,7 @@
 
 @given('I created a new card')
```

Comment:
Since it's a step definition for "cards" it shouldn't be present in "common_steps.py"